src/Autocompletion.ts

```diff
@@ -1,17 +1,26 @@
 import {Job} from "./shell/Job";
 import {leafNodeAt} from "./shell/Parser";
+import {History} from "./shell/History";
+import {Suggestion, styles} from "./plugins/autocompletion_providers/Common";
 
 export const suggestionsLimit = 9;
 
 export const getSuggestions = async(job: Job, caretPosition: number) => {
+    const prefixMatchesInHistory = History.all.filter(line => line.startsWith(job.prompt.value));
+    const suggestionsFromHistory = prefixMatchesInHistory.map(match => new Suggestion({
+        value: match,
+        shouldEscapeSpaces: false,
+        style: styles.command,
```

Comment:
Please create a separate style for history. It used to have a blue icon with a clock: http://fontawesome.io/icon/history/